unified-codes/apps/data-service/src/app/resolvers.ts

```diff
@@ -41,18 +40,11 @@ export const resolvers = {
 
       const { type = 'medicinal_product', code, description, orderBy } = filter ?? {};
       const order = `order${orderBy.descending ? 'desc' : 'asc'}: ${orderBy.field}`;
```

Comment:
Sorry, I know this is from an older PR - but from my prodding of the API, I think it may be nice for orderBy (and the offset/first fields) to either have some default values or to be made mandatory in the schema? I think as it is it's fine for our use case in UI though so doesn't need to be done now necessarily.

Reply:
yes, thanks!
The classes (`EntitySearchFilter` and now `EntitySortInput`) have defaults which covers current usage - I didn't make them mandatory in the schema though, you think I should?

Reply:
I'd say yes just because when they're missing it makes the data-service throw out some not so user-friendly messages (which is kind of fine for us and the web app - but maybe not so much for third parties/other applications later down the line?).

Reply:
good point! thanks.